modules/ROOT/pages/sources-reference.adoc

```diff
@@ -0,0 +1,85 @@
+= Mule Sources
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, flow, source, sources, redelivery, policy
+
+Sources are a very important part of any Mule Application. When added to a flow, they serve as the entry point for information,
+linking any external communication protocol with internal Mule processing.
+
+Examples of sources are:
+
+* xref:docs-connectors::http-connector.adoc[HTTP:Listener]
+* xref:docs-connectors::file-connector.adoc[FILE:Directory Listener]
+* xref:docs-connectors::amqp-connector.adoc[AMQP:Consume]
```

Comment:
AMQP Consume is not a Source

Reply:
You mean that it does not implement Source? 
It is an operation that accepts a redelivery-policy configuration and is used as first MP in a flow right?

Reply:
no, the Consume is a operation, will never be a source.